tests/snippets/attr.py

```diff
@@ -4,12 +4,21 @@
 class A:
     pass
 
+class B:
+    x = 50
```

Comment:
This looks like code from another pull request, is it related to this change, or is it due to git merging?

Reply:
Yeah messed up the rebase